### PR TITLE
Make level configurable for RedirectStdLog

### PR DESCRIPTION
**Pull request by djui**
This change introduces a new function `RedirectStdLogAt`. It has the same behaviour as `RedirectStdLog` but allows providing a log level to be used by used `*log.Logger`.

### CODE REVIEW

File: global.go [hunk 1]
```diff
@@ -128,6 +127,46 @@ func RedirectStdLog(l *Logger) func() {
...
+// standard library's output to os.Stdout.
+func RedirectStdLogAt(l *Logger, level zapcore.Level) (func(), error) {
+	flags := log.Flags()
+	prefix := log.Prefix()
+	log.SetFlags(0)
+	log.SetPrefix("")
+	logger := l.WithOptions(AddCallerSkip(_stdLogDefaultDepth + _loggerWriterDepth))
+	var logFunc func(string, ...zapcore.Field)
+	switch level {
+	case DebugLevel:
```
Comment (djui) [inline global.go]: I can imagine a `levelToFunc` function here to DRY.

**Comment by djui** [comment]
@akshayjshah What do you think?

**Comment by djui** [comment]
@akshayjshah @billf bump.

**Comment by djui** [comment]
@akshayjshah @billf Is there anything I can help out with to get this PR through quicker? :-)

**Comment by akshayjshah** [comment]
No - I'm really sorry for the absurd delay. Family, work, etc. - reviewing now.

File: global.go (hunk 1 above)
Comment (akshayjshah) [reply global.go]: Agreed! You've been patient enough, though - I'm happy to merge without that and fix it myself :)

**Comment by djui** [comment]
@akshayjshah Thanks guys! 🙇 and thanks for the good work on it.

**Pull request merged**